src/main.rs

```diff
@@ -251,3 +251,78 @@ fn find_expression_variables(variables: &mut Vec<String>, expr: &hcl::Expression
         find_expression_variables(variables, &exp);
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{parse_body, Result};
+    use hcl;
+    use std::fs::File;
+
+    fn get_err_contains(block: &hcl::Body) -> String {
+        for attr in block.attributes() {
+            if attr.key.clone().into_inner() == "err_contains" {
+                if let hcl::Expression::String(str) = attr.expr.clone() {
+                    return str;
+                } else {
+                    panic!(
+                        "err_contains expression is not the correct type: {:?}",
+                        attr.expr
+                    )
+                }
+            } else {
+                panic!("Unexpected attribute found: {:#?}", attr);
+            }
+        }
+        String::new()
+    }
```

Comment:
wdyt of making this not panic if we add new attributes to the test case structure - or is the strictness a feature?

something like this maybe?
```
block.attributes.filter_map(|a| {
    if a.key.as_str() == "err_contains" {
        attr.expr.as_str()
     } else {
        None
    }
})
```
not sure about syntax

Reply:
feature. this was a quick implementation, but since no other attribute keys are valid and have no current use I figured it would be better to not allow them to be added